lib/omnibus/software.rb

```diff
@@ -615,7 +615,7 @@ def with_standard_compiler_flags(env = {}, opts = {})
           {
             "LDFLAGS" => "-L#{install_dir}/embedded/lib #{arch_flag}",
             # If we're happy with these flags, enable SSE for other platforms running x86 too.
```

Comment:
Yeah, should fix comment. :)

So this will affect chef-client, which was already released publicly with the sse stuff. We should just be sure this builds on a Windows system before merge.